src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+import style from './Dashboard.module.scss';
+
+const activities = [
+  { activity: 'running', id: 1, timesDone: 3 },
+  { activity: 'swimming', id: 2, timesDone: 2 },
+  { activity: 'not smoking', id: 3, timesDone: 21 },
+];
+
+export const Dashboard = () => {
+  const [formVisible, setFormVisible] = useState<boolean>(false);
+  const [newActivity, setNewActivity] = useState<string>('');
+  const [currentActivities, setCurrentActivities] = useState([...activities]);
+
+  const handleChange = () => {
+    setNewActivity(event.target.value);
```

Comment:
I am even surprised it works here! :D 

It was trying to warn you it is not a good idea
![CleanShot 2023-10-12 at 11 32 45](https://github.com/KamilaSemotanova/tracker/assets/12488648/8c719fa0-c0b6-4032-ba2e-2b20a05834d3)

 You should use the parameter event that you will get from that `onChange` prop. Let's call on this. I will explain how to do it.